tests/test_helpers.py

```diff
@@ -85,6 +85,9 @@
      "above listed projectum ids that you would like to update.\n"
      ),
 ]
+    (["helper", "makeappointments"], "People with gaps in their appointments:\nAnthony Scopatz\nnot fully appointed on 2020-06-15\n"
```

Comment:
this test seems to be not testing much of the intended functionality

Reply:
Yes sorry about that Professor, unfortunately I am not able to push a more comprehensive version of the tests since the `people` schema only has one person with an `appointments` field, and their one appointment is on a grant that is not in the `EXEMPLARS` for grants, so this limits testing ground significantly. I am doing local schema edits to debug and test, but have not pushed these. Would you be open to me making changes to the schema as well, to allow for more testing?

Reply:
yes, you are supposed to modify the exemplars in the schema if you need more tests!    This may (or may not) affect other builders and we will have to deal with that, but better tests are always better than less good tests, and worth the effort in the long run.

Reply:
working on this now that `grant_burn` has been merged!

Reply:
by far the best is to get the `grant_amount_function` functionsmerged then rebase onto updated master.    But if that isn't possible you can merge it in.   Maybe google around on this.  It might be better to squash when you merge and then cherry pick when we merge it into master to avoid that merge, but I am not sure.

Reply:
great, thank you Professor! I will also do some stackoverflow-ing on this